benchmark/benchmain/main.go

```diff
@@ -326,6 +325,38 @@ func readTimeFromInput(values *[]time.Duration, replace string) {
 	}
 }
 
+func partialPrintString(noneEmptyPos []bool, f bm.Features, shared bool) string {
+	s := ""
+	var prefix, suffix string
+	if shared {
+		suffix = "\n"
+	} else {
+		prefix = "-"
+	}
+	if noneEmptyPos[1] {
+		s += fmt.Sprintf("%slatency_%s%s", prefix, f.Latency.String(), suffix)
```

Comment:
Related to the other comment: we should simplify the latency simulation options down to a single "type" enum, instead of individual numbers, and provide 3-5 sets of parameters that may be interesting.
```go
type network struct {
  name string
  latency, kbps, mtu int
}
var networks = []network{{
  "LAN", 2, 100*1024, 1500,
}, {
  "WAN", 30, 20*1024, 1500,
 }, {
  "Longhaul" /* ? name ? */, 200, 1000*1024, 9000,
...
}}
```

Reply:
Done. But in a simple way. Network can be set with one of those values and it overwrites latency/kbps/mtu slices.